src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -211,6 +211,21 @@ export default class GroupComparisonStore {
         invoke:()=>Promise.resolve(this.availableGroups.result!.filter(group=>this.isGroupActive(group)))
     });
 
+    readonly filteredAvailableGroups = remoteData<OverlapFilteredComparisonGroup[]>({
```

Comment:
lets change `filtered` to `nonOverlapping`, it's a bit more specific

Reply:
also lets use this in the body of `availableGroups` in the case `OverlapStrategy.EXCLUDE`